### DISCUSSION TITLE
pagefind + baseUrl = wrong urls in the pagefind index

### DISCUSSION THREAD

**Discussion by indus** [lumeland/lume#502] [Q&A]
my site uses the `baseURL` plugin as well as `pagefind`

with the location `http://myserver.com/some/path/`
the urls in the pagefind index become:
`/some/path/some/path/section/index.html` (`/some/path/` is doubled)

I have not yet found a way to prevent this. Is there anything I can do to fix this?

**Comment by oscarotero** [comment]
Thanks for bringing up this. I think it's fixed, but I cannot release a new version right now. 
Can you update to the latest development version of lume `deno task lume upgrade --dev` and let me know if it works fine. If so, I'll release a new patch this weekend.

**Comment by indus** [comment]
With development version everything works just fine for me 👍 
Thanks for the instant response / fix 🚀

**Reply by oscarotero** [reply]
fixed in Lume v1.19.3
